main.go

```diff
@@ -1,39 +1,64 @@
 package gorillaz
 
```

Comment:
Side note:
We should rename this file, it's confusing to have the name of something runnable in a library

Reply:
yep